packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -389,7 +389,17 @@ LiquidStatement <: Liquid {
 }
 
 LiquidDoc <: Helpers {
-  Node := (TextNode)*
+  Node := (LiquidDocNode | TextNode)*
+  LiquidDocNode =
+    | paramNode
+    | fallbackNode
+
+  fallbackNode = "@" anyExceptStar<newline>
+  paramNode = "@param" space* (paramType)? space* (paramName)? space* "-"? paramDescription?
+  paramType = "{" anyExceptStar<"}"> "}"
+  paramName = identifierCharacter+
+  paramDescription = (~newline identifierCharacter | space | punctuation)+
+  punctuation = singleQuote | doubleQuote | "." | "`"
```

Comment:
This will likely continue to evolve over time as we discover more edge cases. I don't expect this to be perfect, though I want to fix anything smelly now rather than later.

This is one of those scenarios where it's hard to know where I might shoot myself in the foot until the bullet is already in my shoe imo

Reply:
I think we can make paramDescription more permissive - just need to figure out how to do that without bleeding over elsewhere @charlespwd  - any suggestions?

Reply:
Make it grab everything until the end of line? `anyExceptStar<newline | end>` ? (IIRC end is end of string, might be `eof` or something.